h2/src/main/org/h2/res/help.csv

```diff
@@ -6568,6 +6568,8 @@ Returns a JSON object constructed from the specified properties.
 If ABSENT ON NULL is specified properties with NULL value are not included in the object.
 If WITH UNIQUE KEYS is specified the constructed object is checked for uniqueness of keys,
 nested objects, if any, are checked too.
+
+For MySQL and MariaDB compatibility modes, you can also use comma as key/value separator.
```

Comment:
Please, describe this feature here:
https://github.com/h2database/h2database/blob/d05cb83783552cc022b7fad1d965ee06bd5e6149/h2/src/docsrc/html/features.html#L961
https://github.com/h2database/h2database/blob/d05cb83783552cc022b7fad1d965ee06bd5e6149/h2/src/docsrc/html/features.html#L1001
Documentation of `JSON_OBJECT` shouldn't mention mode-specific deviations.